scripts/package-upgrade

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# DESC: Upgrades the Harvester container image
```

Comment:
I think it builds the `rancher/harvester-upgrade` image. Doesn't really upgrade anyything AFAICT.